pkg/hubble/server/serveroption/option.go

```diff
@@ -72,6 +72,8 @@ func WithUnixSocketListener(path string) Option {
 		}
 		if os.Getuid() == 0 {
 			if err := api.SetDefaultPermissions(socketPath); err != nil {
+				socket.Close()
+				unix.Unlink(socketPath)
```

Comment:
another question for my understanding. Any reason to use `golang.org/x/sys/unix` instead of syscall.Unlink ? Thanks.

Reply:
See #10116 for this one.

Reply:
Thanks for your info 💯

I was under impression that x package is kind of experiment only, recently just replaced xerrors package at work.